### PR TITLE
Fix - APP-2553 Refactor to use CSS animation in mobile menu

### PR DESCRIPTION
**Pull request by jamesej**
## Description

Fixing QA issue with original solution for APP-2553, this is a new PR for this ticket
Uses CSS animation and standard events to get same result as framer-motion lib and work with new modal system

Task: [APP-2553](https://aragonassociation.atlassian.net/browse/APP-2553)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2553]: https://aragonassociation.atlassian.net/browse/APP-2553?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/bottomSheet/index.tsx [hunk 1]
```diff
@@ -1,10 +1,7 @@
-import React, {useEffect, ReactNode} from 'react';
-import {motion, PanInfo, useAnimation} from 'framer-motion';
```
Comment (cgero-eth) [inline src/components/bottomSheet/index.tsx:2]: As the `framer-motion` dependency is not used anymore we can remove it from the `package.json` and update the `yarn.lock` file as well

File: src/components/navLink/index.tsx [hunk 2]
```diff
@@ -47,12 +47,12 @@ const NavLink = ({caller, data, onItemClick}: NavLinkProps) => {
   const basePath = pathname.split('/').slice(0, 5).join('/');
   const matches = matchRoutes([{path: data.path}], basePath) !== null;
 
-  const handleOnClick = () => {
+  const handleOnClick = async () => {
     const dao = daoMatch?.params?.dao;
     onItemClick?.();
     // timeout is to allow any state changes triggered by onItemClick to take effect
     // before navigation occurs, potentially unmounting components
-    setTimeout(() => navigate(generatePath(data.path, {network, dao})), 100);
```
Comment (cgero-eth) [inline src/components/navLink/index.tsx:55]: If the `setTimeout` is not needed anymore, we can remove the comment above. Also the function does not need to be `async`, right?

**Review by cgero-eth — COMMENTED** [review]
Thanks @jamesej, tested it and it works as expected now 👌🏻

File: src/components/bottomSheet/index.tsx (hunk 1 above)
Comment (jamesej) [reply src/components/bottomSheet/index.tsx:2]: This is done, thanks

File: src/components/navLink/index.tsx (hunk 2 above)
Comment (jamesej) [reply src/components/navLink/index.tsx:55]: This also done, thanks

**Review by cgero-eth — APPROVED** [review]

**Pull request merged**